docs/AboutUs.adoc

```diff
@@ -3,53 +3,47 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
+CLIndar was developed by the CS2103JAN2018-W13-B3 team. +
 _{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
 {empty} +
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Jarrett Choo
 
-Role: Project Advisor
+{empty}[https://github.com/Kyomian[github]]
+
+Role: To Be Confirmed
```

Comment:
As mentioned before, do add formatting according to the original template =p.
Add in your photos too.